docs/source/json-api/index.rst

```diff
@@ -762,6 +765,7 @@ Where:
 
 - ``templateId`` -- contract template identifier, same as in :ref:`create request <create-request>`,
 - ``key`` -- contract key, formatted according to the :doc:`lf-value-specification`,
+- ``choiceInterfaceId`` -- *optional* template or interface that defines the choice, same format as ``templateId``,
```

Comment:
Same as above.

Reply:
Is there a missing comment above?

Reply:
It was something [along these lines](https://github.com/digital-asset/daml/pull/14410/files#r929969365).